public/index.html

```diff
@@ -9,7 +9,7 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico?v=2">
```

Comment:
it's like this to avoid hitting cache, right?

Reply:
Right. It's cached but from what I've read, in order to make it work, you might have to delete it physically. I've tried both approaches and the `Maker` favicon was still visible